- Created HW3

tzatushevskaya opened the pull request:
- Added images
- Added poisonous plants
- Added a table
- Updated README

Commit:
- Created HW3
 - Added images

eclimov commented:
@tzatushevskaya , please, add reviewers so we could review this pull request

tzatushevskaya commented:
@eclimov done

eclimov commented on public/Sjofn/homework3.html:
```diff
@@ -0,0 +1,100 @@
+        <th style="width: 30%">Description</th>
+    </tr>
+    <tr>
+        <td>I</td>
```
Please, use arabic numerals.

eclimov commented on public/Sjofn/homework3.html:
```diff
@@ -0,0 +1,100 @@
+    <tr>
+        <td>VII</td>
+        <td>Wisteria</td>
+        <td><img src="./assets/wisteria.jpg"></td>
```
You forgot to add the image to VCS (Version Control System)

eclimov replied:
![wisteria_missing_image](https://user-images.githubusercontent.com/16742190/74083115-291cdb80-4a69-11ea-90c8-0ab82aeb5c6c.JPG)

eclimov committed:
- Added image 'bg_pale.png'

eclimov commented on public/Sjofn/homework3.html:
```diff
@@ -0,0 +1,100 @@
+            text-align: center;
+        }
+        tr:not(:first-child):hover {
+            background: url('./assets/bg.jpg') repeat;
```
This image doesn't quite fit here, so I suggest you to use a more pale variant of it.
I added and committed an example for you into your branch: 'bg_pale.png', - try using it instead.

eclimov commented on public/Sjofn/homework3.html:
```diff
@@ -0,0 +1,100 @@
+        }
+        tr:not(:first-child):hover {
+            background: url('./assets/bg.jpg') repeat;
+            opacity: 100%;
```
You don't need this property. It can be removed.

eclimov commented on README.md:
```diff
@@ -54,3 +54,17 @@ Add 2 input fields with placeholders ‘a’ and ‘b’ accordingly.
+        - Description (a short description of the flower)
+    3. Make a ‘zebra’ view of the table
+    4. Add a CSS style that changes background color of a table row’s on hover
+    5. Use palette when picking colors
```
Please, implement this point as well. 
You may pick any palette and set some custom colors to, for example:
- background color (of the table or the whole 'body')
- table's border 
- text
- table's zebra

SandaPanteleimonov reviewed (approved):
Awesome work, background image is stunning!

Commit:
- Changed intelligent rome numbers to primitive arabic ones
 - Added image to git
 - Changed background image
 - Updated README
 - Removed opacity

Commit:
Merge remote-tracking branch 'origin/master' into sjofn-hw3

# Conflicts:
#	README.md

eclimov reviewed (approved):
Approved. Good job!
